Source/Examples/ExampleLibrary/Series/LineSeriesExamples.cs

```diff
@@ -206,7 +206,7 @@ public static PlotModel CustomLineLegendPosition()
             var model = new PlotModel { Title = "LineSeries with LineLegendPosition" };
             model.Axes.Add(new LinearAxis { Position = AxisPosition.Bottom, MinimumPadding = 0.1, MaximumPadding = 0.1 });
             model.Axes.Add(new LinearAxis { Position = AxisPosition.Left });
-            var s1 = CreateExampleLineSeries();
+            var s1 = CreateExampleLineSeries(246);
```

Comment:
Little more white-space before `s1` and `s3` would be nice.

Reply:
I added some white space in this section to make it cleaner.